src/main.cpp

```diff
@@ -246,7 +269,8 @@ void demoRDV()
 
     RdvReader reader;
 
-    auto polyData = vtkPolyData::New();
+    auto polyData =
```

Comment:
meaning is there a specific reason, I am just thinking from the best practices of VTK point of view.